File: generated-sources/kotlin-spring/mojang-authentication/src/main/kotlin/com/github/asyncmc/mojang/authentication/kotlin/spring/model/RefreshRequest.kt
package com.github.asyncmc.mojang.authentication.kotlin.spring.model

import java.util.Objects
import com.github.asyncmc.mojang.authentication.kotlin.spring.model.AccessKeys
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * 
 */
data class RefreshRequest (

        @get:NotNull 
        @JsonProperty("accessToken") override val accessToken: String,

        @JsonProperty("clientToken") override val clientToken: String? = null
) : AccessKeys{

}

File: generated-sources/php-ze-ph/mojang-authentication/src/App/com/github/asyncmc/mojang/authentication/php/ze/ph/model/RefreshRequest.php
<?php

namespace com.github.asyncmc.mojang.authentication.php.ze.ph.server\com.github.asyncmc.mojang.authentication.php.ze.ph.model;

use Articus\DataTransfer\Annotation as DTA;

/**
 */
class RefreshRequest
{
    /**
     * @DTA\Data(field="accessToken")
     * @DTA\Validator(name="Type", options={"type":"string"})
     * @var string
     */
    public $access_token;
    /**
     * @DTA\Data(field="clientToken", nullable=true)
     * @DTA\Validator(name="Type", options={"type":"string"})
     * @var string
     */
    public $client_token;
}
